kerastuner/tuners/hyperband.py

```diff
@@ -51,120 +65,166 @@ class HyperbandOracle(oracle_module.Oracle):
 
     def __init__(self,
                  objective,
-                 max_trials,
+                 max_sweeps,
+                 max_epochs,
```

Comment:
Should this argument have a default value? (probably)

Reply:
I think it might be dangerous to have a default value here:

1) If too low, no models will reach convergence
2) If too high, too many models will be created in each bracket (for example, number of models in bracket 0 are `math.log(max_epochs, factor)`) and also users who forget to do early stopping will have their per-model training times greatly increased

I think it's safer therefore to have the user set it (the default for this before was `10`, which is almost certainly too low for convergence for many NN models)

Reply:
I added a recommendation about how to set this:

```
max_epochs: Int. The maximum number of epochs to train one model. It is
          recommended to set this to a value slightly higher than the expected time
          to convergence for your largest Model, and to use early stopping during
          training (for example, via `tf.keras.callbacks.EarlyStopping`).
```